assets/css/custom.css

```diff
@@ -444,14 +431,21 @@ input[type="text"].letters-input {
   align-items: center;
   flex-direction: row;
   height: 100%;
+  width: 100%;
 }
 
 .bag-color-thumb {
-  height: 50%;
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  height: 100%;
+  width: auto;
+  margin: 0 1%;
 }
 
 .bag-color-thumb img {
   height: 90%;
+  width: auto;
```

Comment:
`width: auto` is the default and shouldn't need to be specified, unless we're overwriting shopify which should go in the overrides section